SingularityUI/app/assets/_index.mustache

```diff
@@ -27,9 +27,9 @@
             hideNewRequestButton: {{{hideNewRequestButton}}},
             defaultCpus: {{{defaultCpus}}},
             defaultMemory: {{{defaultMemory}}},
-            defaultHealthcheckIntervalSeconds: {{{defaultHealthcheckIntervalSeconds}}},
-            defaultHealthcheckTimeoutSeconds: {{{defaultHealthcheckTimeoutSeconds}}},
-            defaultDeployHealthTimeoutSeconds: {{{defaultDeployHealthTimeoutSeconds}}},
+            defaultHealthcheckIntervalSeconds: 5,
+            defaultHealthcheckTimeoutSeconds: 10,
+            defaultDeployHealthTimeoutSeconds: 120,
```

Comment:
i didn't want to amend the commits and force push since that makes things messy but i restored the values in another commit 71b524d058f594da6a878dfef5408435f0b93cd8